examples/hyperparameter_tuning/image_classification_warmstart/hpo.R

```diff
@@ -0,0 +1,117 @@
+#########################################################################
+# This example is an adaptation from
+# https://github.com/awslabs/amazon-sagemaker-examples/blob/master/hyperparameter_tuning/image_classification_warmstart/hpo_image_classification_warmstart.ipynb
+#
+#########################################################################
+
+#########################################################################
+# Packages required
+#########################################################################
+library(R6sagemaker)
+library(data.table)
+library(fastDummies)
+
+#########################################################################
+# NOTE:
+#
+# R6sagemaker by default sets logging level to INFO.
+# If you wish to change the level for the logging level please use the
+# logger package, for example:
+#
+# >>> logger::log_threshold(logger::DEBUG)
+#########################################################################
+
+#########################################################################
+# Setup 
+#########################################################################
+sess = Session$new()
+region = sess$paws_region_name
+role = get_execution_role(sess)
+bucket = sess$default_bucket()
+prefix = 'ic-fulltraining'
+
+# To do!!!!
+training_image = get_image_uri(sess.boto_region_name, 'image-classification', repo_version="1")
```

Comment:
You are right, I suppose I should have put it the other-way around. Line 53 should be removed in this instance and line 34 kept in its place.  Although I kept it in the 'To do!!!' because of the sess.boto_region_name variable I wasn't sure how to translate across.